### ISSUE TITLE
Predictions need to be their own tab.  They are not a typing platform!

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1833]
![image](https://user-images.githubusercontent.com/7661533/34229678-ca051020-e59a-11e7-827c-8889107cbec5.png)

**Comment by JoshuaVSherman** [comment]
@dmachi 
Found where this is, at http://www.patric.local:3000/view/GenomeList
but i'm not sure how to reproduce the information being presented

**Comment by mshukla1** [comment]
Please don’t change anything here, it requires some more thought. 

I will come up with a recommendation. 

-Maulik

**Comment by rkenyon** [comment]
Jim triage comment: Maulik has been working on this because he showed me some mock ups of what it might look like.  It is a bad problem until it finally gets fixed.

**Comment by mshukla1** [comment]
This has been addressed with the recent changes to the solr genome_amr core and related changes to the AMR phenotype table on the website. 

We have added new fields for 
evidence 
pmid
computational method
computational method version 
computational method performance. 

The data related to predictions have been table out from laboratory method * fields and populated in computational method related fields. 

There is a facet / filter now to easily allow users to filter and view AMR phenotypes from either laboratory or computational methods. 

<img width="1366" alt="Screen Shot 2019-04-07 at 10 35 09 AM" src="https://user-images.githubusercontent.com/1236463/55685911-db9f8180-5920-11e9-9425-d8437922d628.png">
